reference/features/search_parameters.md

```diff
@@ -368,17 +368,26 @@ You would get the following response with **information about the matches in the
 {
   "id": "50393",
   "title": "Kung Fu Panda Holiday",
-  "poster": "https://image.tmdb.org/t/p/w1280/gp18R42TbSUlw9VnXFqyecm52lq.jpg",
+  "poster": "https://image.tmdb.org/t/p/w500/rV77WxY35LuYLOuQvBeD1nyWMuI.jpg",
```

Comment:
Not sure if I should update this link.

Reply:
Wow, wouldn't have even thought of doing that! Thanks 😊